### PR TITLE
Decouple pivot and re-pivot actions

### PR DESCRIPTION
**Pull request by kashifest**
This PR separates pivoting and re-pivoting into separate actions.

### CODE REVIEW

**Comment by furkatgofurov7** [comment]
Centos job is also ready for feature tests, maybe we have to run it, as well? `/test-features-centos`

**Comment by kashifest** [comment]
> Centos job is also ready for feature tests, maybe we have to run it, as well? `/test-features-centos`

I believe if once of them is running ok, thats enough, since this has nothing to do with node image OS

File: vm-setup/roles/v1aX_integration_test/tasks/main.yml [hunk 1]
```diff
@@ -51,6 +51,10 @@
 
 - name: pivot
   include: move.yml
+  when: v1aX_integration_test_action == "pivoting" or v1aX_integration_test_action == "upgrading"
```
Comment (fmuyassarov) [inline vm-setup/roles/v1aX_integration_test/tasks/main.yml:54]: Are you expecting to use move.yml sometime during the upgrade test?

**Review by smoshiur1237 — COMMENTED** [review]
looks good to me. lgtm

**Review by fmuyassarov — COMMENTED** [review]
/lgtm

**Comment by fmuyassarov** [comment]
> /test-centos-integration

I don't understand what's wrong with the centos CI.

**Comment by kashifest** [comment]
> > /test-centos-integration
> 
> I don't understand what's wrong with the centos CI.

In one of the logs I saw aroun 16 IPMI call fails over the span of 40 minutes

**Review by maelk — APPROVED** [review]

**Pull request merged**